google/cloud/aiplatform/metadata/metadata.py

```diff
@@ -111,5 +203,50 @@ def _validate_experiment_and_run(self, method_name: str):
                 f"No run set. Make sure to call aiplatform.set_run('my-run') before trying to {method_name}. "
             )
 
+    def _query_runs_to_data_frame(
+        self, context_name: str, context_resource_name: str, source: str
+    ) -> pd.DataFrame:
+        filter = f'schema_title="{constants.SYSTEM_RUN}" AND in_context("{context_resource_name}")'
```

Comment:
here should be SYSTEM_PIPELINE_RUN for the other one?

Reply:
Actually no.  SYSTEM_PIPELINE_RUN is a context scheme in stead of execution schema. basically under each system.pipeline, there is a SYSTEM_PIPELINE_RUN and a system.Run, the pipelineRun would store the DAG graph, and the Run will more or less be a shortcut to store param and connect to metric artifacts. Dushyanth's doc would have some more info although I think some edges was omitted in his doc.

Reply:
ack, SG